README.md

```diff
@@ -134,3 +134,13 @@ pyenv install 2.7.18
 pyenv global 2.7.18
 echo 'PATH=$(pyenv root)/shims:$PATH' >> ~/.zshrc
 ```
+
+With the latest changes, we might run into a situation where the local strapi does not have the ifixit test store in it. In order to fix this add the folowing to `backend/.env`
+
+```
+STRAPI_ADMIN_ENABLE_ADDONS_DANGEROUS_ACTIONS=true
+```
+
+With a recent version of main, go to your local strapi at http://localhost:1337/admin/plugins/addons and hit `Reset Seed` with the domain set to `main.govinor.com`.
```

Comment:
I would just say "latest"

Reply:
That is definitely better. Thanks

Reply:
Resolved d6db2d016e9d11e0ad95877d71e2debddf28bb99